src/System.IO.FileSystem.DriveInfo/src/System/IO/DriveInfo.BSD.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
Again we are not confident about having a generic "BSD" code path. This should either go to the Unix folder or the specific FreeBSD one.